pkg/external/funcs.go

```diff
@@ -56,3 +61,52 @@ func GetKeystoneAPI(
 
 	return &keystoneList.Items[0], nil
 }
+
+//
+// GetAdminServiceClient - get an admin serviceClient for the keystoneAPI instance
+//
+func GetAdminServiceClient(
+	ctx context.Context,
+	h *helper.Helper,
+	keystoneAPI *keystonev1.KeystoneAPI,
+) (*gophercloud.ServiceClient, condition.Condition, ctrl.Result, error) {
+	// get public endpoint as authurl from keystone instance
+	authURL, err := keystoneAPI.GetEndpoint(common.EndpointPublic)
+	if err != nil {
+		return nil, condition.Condition{}, ctrl.Result{}, err
+	}
+
+	// get the password of the admin user from the
```

Comment:
Looks like this comment is missing its ending

Reply:
done, updated the comment